### ISSUE TITLE
Admin Theme Media Button Issues

### ISSUE THREAD

**Issue by jcsparks** [CuBoulder/tiamat-theme#689]
Two UI issues seem to have been introduced recently when dealing with media insertion... 

First it looks like in the context where there's an option to choose between an Image and a Video the buttons on the left are just links and lost their background color and padding: 
<img width="746" alt="Screenshot 2024-02-20 at 4 16 18 PM" src="https://github.com/CuBoulder/tiamat-theme/assets/12704433/03ee66c6-141a-45df-99ff-8cab828df42c">

Second the button to add an image and select an image is getting a `background : none` setting from one of the modules which is causing the button to visually disappear with super low contrast text: 

<img width="1262" alt="Screenshot 2024-02-20 at 4 17 46 PM" src="https://github.com/CuBoulder/tiamat-theme/assets/12704433/dcb9d319-e02a-4daa-afba-cc36dd0e746c">

**Comment by timurtripp** [comment]
I thought I'd been able to reproduce the transparent button problem but I can't.

**Comment by timurtripp** [comment]
The one I was able to reproduce is the missing padding. It's showing an undefined CSS variable.
